spotbox/spotboxgui.py

```diff
@@ -191,6 +191,8 @@ def _loadspot(self, spottext, spotnumber):
             self.spot_durations[spotnumber] = self._menus.filetimetoload
 
     def _playspot(self, spotnumber):
+        self._stop()
```

Comment:
Since iTunes only has one playback queue, playing a new spot will stop playing that spot and start playing the new spot.

With the Pyglet player, it's possible to play multiple spots simultaneously, which is probably not what most people expect. Adding an explicit call to `stop` doesn't affect iTunes playback but explicitly implements behavior that was a side effect of iTunes mode.